queries/superhtml/highlights.scm

```diff
@@ -0,0 +1,47 @@
+(doctype) @constant
+
+(comment) @comment
+
+(tag_name) @tag
+
+((tag_name) @string.special
+  (#any-of? @string.special "super" "extend"))
+
+(attribute_name) @attribute
+
+((element
+  (start_tag
+    (attribute
+      (attribute_name) @attribute
+      [
+        (attribute_value) @markup.link.url
+        (quoted_attribute_value
+          (attribute_value) @markup.link.url)
+      ]))
+  (element
+    (start_tag
+      (tag_name) @tag)))
+  (#eq? @tag "super")
+  (#eq? @attribute "id"))
+
+((element
+  (start_tag
+    (tag_name) @string.special
+    (attribute
+      (attribute_name) @attribute)+))
```

Comment:
so is this. don't need to set `@attribute` since your general `(attribute_name) @attribute` query up above already cover it